content/en/blog/2023/http-conventions-declared-stable.md

```diff
@@ -0,0 +1,264 @@
+---
+title: HTTP semantic conventions are declared stable
+linkTitle: HTTP semantic conventions
+date: 2023-11-03
+author: '[Trask Stalnaker](https://github.com/trask) (Microsoft)'
+cSpell:ignore: stalnaker trask
+---
+
+Earlier this year we embarked on a three month project to stabilize HTTP semantic conventions.
+
+We are proud to announce (nine months later), that the HTTP semantic conventions are now the first OpenTelemetry semantic convention to be declared stable!
+
+This stable version is a significant evolution over the previous unstable version.
+
+Because of the number of changes, and because of the number of users impacted by these changes, we have mapped out
+a transition plan for instrumentations to follow when updating to the stable version which prioritizes our users.
+
+We also want to share some of the reasons behind these changes, as we think that the stable version brings many
+significant improvements which will make the pain of this upgrade worth it in the long run.
+
+## Aligning with other open standards
```

Comment:
This and the other level-2 headings below don't quite make sense as separate sections. Maybe they need to be list bullets? If I understand their purpose, I think that these are meant to be named reasons for the HTTP changes? If so, that needs to be made clear.

Reply:
ya, thx, I agree, I reduced it to a more simplified list of bullets